### PR TITLE
LPS-74426 source formatting

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

File: modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+ * @author Christopher Bryan Boyd
+ */
+public class BuildCSSMojoTest {
+
+	@ClassRule
+	public static final MavenExecutor mavenExecutor = new MavenExecutor();
+
+	@BeforeClass
+	public static void setUpClass() throws IOException {
+		final String mavenGoalCssBuilder = "css-builder:build";
```
Comment (gamerson) [inline modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java:47]: Can you try to move all of these variables here to global private final static?  I think that should be possible.

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+		final String mavenRepoArgument = String.format(
+			"-Dmaven.repo.local=%s", _repoPath);
+		final String mavenVersionArgument = String.format(
+			"-Dcssbuilder.version=%s", _version);
+		final String mavenRepoLocationArgument = String.format(
+			"-Dcssbuilder.repolocation=%s", _repoPath);
+
+		_goodMavenCommand = new String[] {
```
Comment (gamerson) [inline modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java:55]: Can you change the variables 'goodXXX' and 'badXXX' to something more like 'broken' and 'fixed'?

File: modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+		final boolean anyCssFilesInDirectory = _anyCssFilesInDirectory(
+			_tempCss.toPath());
+
+		Assert.assertFalse(anyCssFilesInDirectory);
+	}
+
+	@Test
+	public void testGoodPom() throws Exception {
+		System.out.println("Before execute maven");
```
Comment (gamerson) [inline modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java:102]: go ahead and remove the sysouts.

File: modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+		_executeMaven(_tempPomFolder.toPath(), _goodMavenCommand);
+		System.out.println("After execute maven");
+		final boolean anyCssFilesInDirectory = _anyCssFilesInDirectory(
+			_tempCss.toPath());
+
+		Assert.assertFalse(anyCssFilesInDirectory);
+	}
+
+	@Rule
```
Comment (gamerson) [inline modules/util/css-builder/src/test/java/com/liferay/css/builder/maven/BuildCSSMojoTest.java:111]: Put this rule uptop with the other one

**Comment by christopherbryanboyd** [comment]
Pull request submitted to https://github.com/gamerson/liferay-portal/pull/25. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**